spec/models/member_spec.rb

```diff
@@ -50,6 +50,27 @@
     end
   end
 
+  describe 'validations' do
+    describe 'email' do
+      before do
+        create(:member, email: 'foo@example.com')
+      end
+
+      it 'must be unique' do
+        member = build(:member, email: 'foo@example.com')
+        expect(member).to be_invalid
+        expect(member.errors[:email]).to eq(["has already been taken"])
```

Comment:
what about blank emails? what if the form doesn't ask for email, can there only be one nil email? would love to see some specs for more of those cases